include.js

```diff
@@ -964,6 +964,13 @@ let cards = [
     author: 'Jeet Kanodia',
     githubLink: 'https://github.com/jeetkanodia'
   },
+  {
+    artName: 'Blob Scale',
+    pageLink: './Art/romankyv/index.html',
+    imageLink: './Art/romankyv/BlobScale.gif',
+    author: 'Roman',
+    githubLink: 'https://github.com/romankyv'
+  },
```

Comment:
# Case in your path
Please adjust the paths according to your folder name:
- your folder name starts with a capital whereas your paths for your contribution here - `./Art/romankyv` 
--> romankyv: start with a lowercase